sse2neon.h

```diff
@@ -25,6 +25,7 @@
 //   Yang-Hao Yuan <yanghau@biilabs.io>
 //   Syoyo Fujita <syoyo@lighttransport.com>
 //   Brecht Van Lommel <brecht@blender.org>
+//   Remi Arnaud <remi@acm.org>
```

Comment:
Use `Co-authored-by` information in git commit messages instead. See https://docs.github.com/en/github/committing-changes-to-your-project/creating-a-commit-with-multiple-authors

Reply:
Done.